pkg/subctl/cmd/show_connections.go

```diff
@@ -0,0 +1,99 @@
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	submarinerclientset "github.com/submariner-io/submariner-operator/pkg/client/clientset/versioned"
+	"github.com/submariner-io/submariner-operator/pkg/subctl/operator/submarinercr"
+	submv1 "github.com/submariner-io/submariner/pkg/apis/submariner.io/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type connectionStatus struct {
+	gateway     string
+	cluster     string
+	remoteIp    string
+	cableDriver string
+	subnets     string
+	status      submv1.ConnectionStatus
+}
+
+func newConnectionsStatusFrom(gateway string, cluster string, remoteIp string, cableDriver string, subnets string, status submv1.ConnectionStatus) connectionStatus {
```

Comment:
Same comment as in the other patch, it's cleaner to create the structure right away, unless you had some business logic for creating the structure.